core/client/app/templates/setup/three.hbs

```diff
@@ -12,7 +12,8 @@
 sally.sanders@example.com" value=users}}
     </form>
 
-    <button {{action "signin"}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
+    <button {{action 'invite'}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
         {{buttonText}}
     </button>
+    <a {{action 'signin'}} href="#" class="gh-flow-skip">I'll do this later, take me to my blog!</a> 
```

Comment:
Sorry, I didn't mean to be so vague. I thought you had 2 elements on the page and were swapping them based on state, which seemed odd. However, I now realise they're actually both on the page at the same time, which is definitely not the right thing. They should be the same button as they were before.